scheduler/src/cook/scheduler/scheduler.clj

```diff
@@ -582,18 +582,24 @@
             (when is-rate-limited?
               (swap! user->rate-limit-count update user #(inc (or % 0))))
             (not (and is-rate-limited? enforcing-job-launch-rate-limit?))))
-        considerable-jobs
-        (->> pending-jobs
-             (util/filter-based-on-quota user->quota user->usage)
-             (filter (fn [job] (util/job-allowed-to-start? db job)))
-             (filter user-within-launch-rate-limit?-fn)
-             (filter launch-plugin/filter-job-launches)
-             (take num-considerable)
-             ; Force this to be taken eagerly so that the log line is accurate.
-             (doall))]
+        quota-filter-jobs (util/filter-based-on-quota user->quota user->usage pending-jobs)
+        allowed-to-start-jobs (filter (fn [job] (util/job-allowed-to-start? db job)) quota-filter-jobs)
+        within-launch-rate-jobs (filter user-within-launch-rate-limit?-fn allowed-to-start-jobs)
+        launch-plugin-jobs (filter launch-plugin/filter-job-launches within-launch-rate-jobs)
```

Comment:
What does this filter do? The name `launch-plugin-jobs` doesn't tell the reader anything about what's happening.

Reply:
If we're going to give it a name, let's give it a meaningful name